mantidproject/mantid
# Fixed Engineering gui ignoring browsed to file

**Description of work.**
The engineering gui now correctly loads the file selected via browse, rather than trying to find a valid run number contained in any folder along the path to the selected file.

**To test:**
Interfaces->Diffraction->Engineering
enter 1 as the RB number, ensure the input directories in the settings tab are set, then head to the calibration tab, set the logger level to information,click browse on the vanadium and select the enginx data provided in the systemtests (SystemTest/ENGINX00236516, and 241391), the logger should show that it has found the correct path, do the same on the calibration, then select calibrate, checking the history the correct path should show there, rename one of the files and then try again it should use the newly renamed file, and not got stuck trying to find a number in the folder in question, repeat this with the focus tab and ENGINX_280625_focused_bank_1.

Fixes #25439 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
